docs/source/release_notes/release-notes.rst

```diff
@@ -43,42 +43,45 @@ develop
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1872>`__)
-
-    *    - |fixed|
-         - Fixed a bug that would cause console to error on any range request that used a column selection and had more than one batch of results.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1876>`__)
+    *    - |devbreak| |improved|
+         - The format of serialized exceptions occurring on a remote host has been brought in line with that of the `palantir/http-remoting <https://github.com/palantir/http-remoting>`__ library.
+           This should generally improve readability and also allows for more meaningful messages to be sent; we would previously return message bodies with no content for some exceptions (such as ``NotCurrentLeaderException``).
+           In particular, the assumption that a status code of 503 definitively means that the node being contacted is not the leader is no longer valid.
+           That said, existing AtlasDB clients will still behave correctly even with a new TimeLock.
+           (`Pull Request 1 <https://github.com/palantir/atlasdb/pull/1831>`__,
+           `Pull Request 2 <https://github.com/palantir/atlasdb/pull/1808>`__)
 
     *    - |new|
          - Timelock server now has jar publication in addition to dist publication.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1898>`__)
 
-    *    - |improved| |devbreak|
-         - The format of serialised exceptions occurring on a remote host has been brought in line with that of the `palantir/http-remoting <https://github.com/palantir/http-remoting>`__ library.
-           This should generally improve readability and also allows for more meaningful messages to be sent; we would previously return message bodies with no content for some exceptions (such as ``NotCurrentLeaderException``).
-           In particular, the assumption that a status code of 503 definitively means that the node being contacted the leader is no longer valid. That said, existing AtlasDB clients will still behave correctly even with a new TimeLock.
-           (`Pull Request 1 <https://github.com/palantir/atlasdb/pull/1831>`__,
-           `Pull Request 2 <https://github.com/palantir/atlasdb/pull/1808>`__)
-
     *    - |new| |fixed|
          - TimeLock clients may now receive a ``BlockingTimeoutException`` 503 if they make a lock request that blocks for longer than the server's idle timeout.
            Previously, these requests would be failed with a HTTP-level exception that the stream was closed.
            We have rewritten clients constructed via ``AtlasDbHttpClients`` to account for this new behaviour, but custom clients directly accessing the lock service may be affected.
            This feature is disabled by default, but can be enabled following the TimeLock server configuration :ref:`docs <timelock-server-time-limiting>`.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1808>`__)
 
+    *    - |improved|
```

Comment:
Agreed, I've bumped it down 2 places - that sounds like a good addition to the heuristic as well (i.e. items that are likely to interest _only_ developers should be at the end).